### DISCUSSION TITLE
Public Forms

### DISCUSSION THREAD

**Discussion by rjsandim** [filamentphp/filament#12084] [Help]
### Package

Form builder

### Package Version

Lastest

### How can we help you?

Is it possible to let a for public and shareable and let people fill them and  use it like google forms? 

Or always must be used by logged in users? 

If is it possible how do I let it public?

**Comment by VincvanAs** [comment]
Maybe this could help you out

https://github.com/filamentphp/filament/discussions/8843#discussioncomment-7178537

Combined with maybe: https://filamentphp.com/docs/3.x/forms/adding-a-form-to-a-livewire-component
